Make use of the new builder

vicary opened the pull request:
Because `ng-model=“$$value$$”` doesn’t work on arrays.

scottux commented:
I have done the same work and also upgraded dependencies, and managed to get the tests working https://github.com/scottux/angular-schema-form-datepicker

vicary commented:
@scottux I don't really have the time to keep-up with core changes in the past year. If you don't mind picking up I can address a few more changes in order to merge with the current development branch in core.

Taking references from https://github.com/json-schema-form/angular-schema-form-bootstrap/blob/develop/src/bootstrap/default.html, they
1. added `"::"` to most of the one-time bindings, and
2. replaced `form.key.slice(0, -1)` with `fieldId(true, false)`.
